x-pack/qa/full-cluster-restart/src/javaRestTest/java/org/elasticsearch/xpack/restart/FullClusterRestartIT.java

```diff
@@ -556,6 +556,7 @@ public void testSlmPolicyAndStats() throws IOException {
             "* * * 31 FEB ? *",
             "test-repo",
             Collections.singletonMap("indices", Collections.singletonList("*")),
+            null,
             null
```

Comment:
Nit: this is the only change in this file but it's not really necessary since we're keeping the delegate constructor because there are too many usages anyway, so I'd also be in favor of reverting this change.